src/Player.java

```diff
@@ -33,6 +36,7 @@ public int CharMenu(){
         System.out.println( "1 - " + " Type = Samurai   Damage = 5   Healty = 21   Money = 15");
         System.out.println( "2 - " + " Type = Archer    Damage = 7   Healty = 18   Money = 20");
         System.out.println( "3 - " + " Type = Knight    Damage = 8   Healty = 24   Money = 5");
+        System.out.println( "4 - " + " Type = Mustapha  Damage = 3   Healty = 35   Money = 65");
         System.out.println("Your Choice");
         int charID = scan2.nextInt();
         while (charID<1 || charID>3){
```

Comment:
case number should be 4

Reply:
ahhh, you re right. changing in new co mmit